packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -26,6 +28,17 @@ This guide provides a working example of an Okta Event Hook. It uses the ngrok u
 
 ---
 
+## Overview
+
+Event Hooks are outbound calls from Okta that can notify your own software systems of events occurring in your Okta org. The ngrok utility exposes local applications to the internet and receives and responds to Event Hook calls. This setup enables the development and testing of Event Hooks locally, rather than using an external test or production environment. The ngrok utility also provides replay functionality that assists in testing and developing your external service code.
+
+To use this guide:
+
+* Create a local application or use an existing application.
+* Run the ngrok utility with your local application.
+* Create an Event Hook.
+* Preview and test the Event Hook.
+
```

Comment:
I'm not sure if it's the convention to use bullets? If these steps need to be executed in order, perhaps an ordered list  is better suited?

Reply:
It's the way I presented it in the guide, but in theory you could create a hook first. Anyway, I added numbering. I didn't want to add links to the sections; I thought it'd be overkill,  but I still like the idea of laying out the high-level tasks at the start. Thanks.